guides/source/active_record_callbacks.md

```diff
@@ -266,7 +545,8 @@ Book/Library was touched
 => true
 ```
```

Comment:
Let's update these examples slightly: use `user` instead of just `u` to keep it consistent with other examples, and use `book` instead of `@book`, no need to use instance vars in an irb/console session.

Reply:
✅